### DISCUSSION TITLE
SmallRecords list import error

### DISCUSSION THREAD

**Discussion by cdurbrow** [wintercms/winter#662] [Q&A] [answered]
I am new to this CMS. I have installed Winter CMS current version Mysql. I have installed one October plugin JanVince.SmallRecords.
I understand this is an October plugin and that things may eventually break.

There seems to be an error in the Small records > Records lists > import records.

debug shows
```
A syntax error was detected in /var/www/html/wntcms/plugins/janvince/smallrecords/models/record/columns_import.yaml. Duplicate key "type" detected at line 11 (near "'type': text") at line 68 (near "/var/www/html/wntcms/vendor/winter/storm/src/Parse/Yaml.php").
```

commenting out the lines from 'Slug' to the end of 'Name', did resolve the error but caused an issue with the 'Name' field.

Funny thing, import works in Catagories.

I figured I would start the conversation here. Has anyone used this plugin or can recommend a Winter plugin for managing lists that hase import export features.

**Comment by RomainMazB** [comment, marked as answer]
It looks like the plugin does contain syntax errors: https://github.com/jan-vince/smallrecords/blob/master/models/record/columns_import.yaml#L10-L18 (the line `type: text` should appear only once per field)

Winter's members won't support a plugin maintained by an octobercms' partner (I suggest that you read this to understand why: https://wintercms.com/blog/post/we-have-forked-october-cms)

**Reply by cdurbrow** [reply]
Thank you so much for the quick reply. I have removed the double type: text from my local copy and the import error is gone. I will continue testing this plugin. I understand about not supporting 3rd and 4th party code.

**Reply by LukeTowers** [reply]
I submitted a PR with the fix, thanks @RomainMazB, see https://github.com/jan-vince/smallrecords/pull/98. No idea if @jan-vince still maintains the plugin though.

**Reply by jan-vince** [reply]
Sure, I will fix that. Thanks for the PR @LukeTowers.

**Reply by LukeTowers** [reply]
No problem, thanks! 😄
